src/lib/helpers/date.ts

```diff
@@ -73,6 +73,53 @@ export const toLocaleTimeISO = (datetime: string | number) => {
     return date.toLocaleTimeString('sv');
 };
 
+export const utcHourToLocaleHour = (utcTimeString: string) => {
```

Comment:
would you please add some test cases for the new methods? The tests assume [EST](https://github.com/appwrite/console/blob/afb50f720d94d0d92a4807d1931624fc55b41386/package.json#L15).

Reply:
Ahh sure, will do that.